edubots_surveys.Rnw

```diff
@@ -466,54 +466,52 @@ We will present next the methodology we followed to find out teachers' opinions.
 \section{Methodology}
 \label{sec:meth}
 
-This study has employed a quantitative approach in order to address its research objective. A good amount of data was collected on the usage of messaging applications and chatbots by educators in universities and colleges.\footnote{In Spain, higher education institutions generally known elsewhere as colleges, are called simply high schools, and deliver
-tertiary or associate degrees that do not have the consideration of university degrees.}
-To achieve this goal two online surveys were designed and developed using Google Forms
-in Spanish. They are comprised of six questions focusing on demographic data
-(i.e., sector, gender, degrees, discipline, age, teaching experience)
-followed by several multiple choice questions allowing the participants to choose multiple answers.
-
-In the {\bf first survey}, the multiple choice questions are
-focused on the use of messaging apps in the teaching practice, the type of chatbot use cases that teachers would consider
-useful for their teaching and the impact of COVID on the teaching practice; please check them out to the full extent in the Appendix.\footnote{This {\bf first survey} was made in two separate Google forms, one for
-university and the other for teachers in tertiary education institutions, outside the university system (vocational training, including 1- and 2-year degrees). Questions and
+This study employed a quantitative approach to address its research objectives. A significant amount of data was collected on the usage of messaging applications and chatbots by educators in universities and colleges.\footnote{In Spain, higher education institutions that would generally known as colleges in other places, are simply called high schools, and provide
+tertiary or associate degrees that do not have the same consideration as university degrees.}
+To collect this data, two online surveys were designed and developed using Google Forms
```

Comment:
I wouldn't leave the word "goal" out entirely. Data was gathered to try and achieve one goal.

Reply:
I'm sorry, i don't see it in the excerpt. Just go and change it if you feel like it